app/assets/javascripts/float-kit/addon/modifiers/close-on-click-outside.js

```diff
@@ -12,7 +12,12 @@ export default class FloatKitCloseOnClickOutside extends Modifier {
     this.closeFn = closeFn;
     this.trigger = trigger;
     this.element = element;
-    document.addEventListener("click", this.check, {
+
+    document.addEventListener("pointerdown", this.check, {
+      passive: true,
+    });
+
+    document.addEventListener("touchstart", this.check, {
```

Comment:
this is the core of the fix

Reply:
Is there a need for `touchstart` here? As I understand it`, pointer events will fire for both mouse and touch devices?

Reply:
wow 🤯  I was pretty sure it was not the case, TIL, nice.